autosubmit/job/job.py

```diff
@@ -2698,6 +2644,16 @@ def recover_last_ready_date(self) -> None:
                     self.ready_date = datetime.datetime.fromtimestamp(stat_file.stat().st_mtime).strftime('%Y%m%d%H%M%S')
                     Log.debug(f"Failed to recover ready date for the job {self.name}")
 
+decorator_list: list = []
+list_decorators = ['x11', 'x11_options', 'tasktype', 'jobname', 'script', 'fail_count', 'retrials', 'checkpoint',
+                    'sdate', 'member', 'chunk', 'split', 'delay', 'wallclock', 'hyperthreading', 'nodes', 'numthreads',
+                    'threads', 'cpus_per_task', 'numtask', 'tasks', 'tasks_per_node', 'scratch_free_space', 'memory',
+                    'memory_per_task', 'frequency', 'synchronize', 'dependencies', 'delay_retrials', 'packed', 'export',
+                    'custom_directives', 'splits', 'notify_on', 'status', 'log_recovered', "current_queue", 'numproc',
+                    'processors', 'processors_per_node']
+for decorator_to_import in list_decorators:
+    decorator = Job(name=decorator_to_import)
+    decorator_list.append(decorator.__doc__)
```

Comment:
Note to self: check why this was added.

Reply:
Hmm, but where is it used after you create the `decorator_list`? I think the replacement for the decorator is more or less following this idea. But you still have the `autosubmit_parameters`. That can be removed too, I think.

This part I think still needs some further work, but maybe we can work together to finish it for 4.1.16, or bump it to 4.1.17 (I think you have nearly 80% of the work done here, which is why I think there might still be time for 4.1.16).